operators/src/upgrade/helm/client.rs

```diff
@@ -104,13 +159,12 @@ impl Chart {
             .arg("--output")
             .arg("json");
 
-        let output = command.output().map_err(|_| {
-            Error::HelmStdError("Error while running helm ls command".to_string())
-        })?;
+        let output = command
```

Comment:
Above, can we make use of our type instead of calling the binary directly?
```rust
HelmArgs::default()
            .with_name(None)
            .with_namespace(self.chart.namespace.clone())
            .run(["ls", "--filter", exact_match.as_str()])?;
```
Ah in this case we wouldn't have the name in the command? Maybe name should then be an option?